src/pyosmeta/parse_issues.py

```diff
@@ -221,6 +221,25 @@ def _postprocess_meta(self, meta: dict, body: List[str]) -> dict:
 
         return meta
 
+    def _postprocess_labels(self, meta: dict) -> dict:
+        """
+        Handle specific labels that are model properties
+        """
+
+        def _is_archived(label: str) -> bool:
```

Comment:
this is a super explicit name but i think for the purposes of documenting process, over-documenting via docstrings is a good idea here. so let's add numpy docstrings with very clear descriptions of intent to each of these.

Reply:
I added a docstring in b9868eb